services/web/client/source/class/osparc/auth/Manager.js

```diff
@@ -148,6 +148,8 @@ qx.Class.define("osparc.auth.Manager", {
     __loginUser: function(profile) {
       osparc.auth.Data.getInstance().setEmail(profile.login);
       osparc.auth.Data.getInstance().setToken(profile.login);
```

Comment:
please just double-check the openapi specs. I think you guys removed the entry "login" in the profile

Reply:
still there